prow/jobs/test-infra/test-job-definitions.yaml

```diff
@@ -1,7 +1,7 @@
 presubmits: # runs on PRs
   kyma-project/test-infra:
   - name: test-infra-test-jobs-yaml-definitions
-    run_if_changed: "(prow/jobs)|(development/tools/jobs)"
+    run_if_changed: "^(prow/jobs/|prow/config\\.yaml$|development/tools/jobs/)"
```

Comment:
I think there is a typo in the regex:

![zrzut ekranu 2018-12-20 o 10 36 11](https://user-images.githubusercontent.com/13346689/50276812-244f2f80-0443-11e9-9459-fa0320b29a60.png)

`^(prow/jobs/|prow/config\.yaml$|development/tools/jobs/)` One `\` too many :)

![zrzut ekranu 2018-12-20 o 10 38 40](https://user-images.githubusercontent.com/13346689/50276914-7001d900-0443-11e9-8cba-efceac21812b.png)

Reply:
No typo, it is ok. `\` is required, because in other case `.` will be not escaped for regex formule.  So `\\` will be changed to `\` when string will be parsed 😄